help.go

```diff
@@ -120,21 +120,21 @@ func DefaultHelpPrinter(options HelpOptions, ctx *Context) error {
 	if ctx.Empty() {
 		options.Summary = false
 	}
-	w := newHelpWriter(ctx, options)
+	w := NewHelpWriter(ctx, options)
 	selected := ctx.Selected()
 	if selected == nil {
-		printApp(w, ctx.Model)
+		PrintApp(w, ctx.Model)
 	} else {
-		printCommand(w, ctx.Model, selected)
+		PrintCommand(w, ctx.Model, selected)
 	}
 	return w.Write(ctx.Stdout)
 }
 
-func printApp(w *helpWriter, app *Application) {
+func PrintApp(w *HelpWriter, app *Application) {
```

Comment:
Is it going to be useful to expose these `Print*` functions? Unless you want segments in exactly the same format, I'm not sure it will be that useful. WDYT?

Reply:
Yeah. I didn't want to flood the public functions but the format of these functions is a bit confusing. My main use case is I want to get the flags output and the arguments output without anything else. Here is the output i'm going for and I would like to get the sections under ARUGMENTS and FLAGS with a function call instead of trying to manipulate the stdout.
```
─ sq ski:debug:app --help                                                      ─╯
USAGE
   sq ski:debug:app <app-name> --env <ENV> --region <REGION>
   sq ski:debug:app -a <app-name> --env <ENV> --region <REGION>

ARGUMENTS
   [<app-arg>]

FLAGS
   -h, --help             Show context-sensitive help.
       --summary
   -a, --app=STRING
   -e, --env=STRING       environment i.e. dev, development, prod, production
   -r, --region=STRING    region i.e. west, us-west-2, east, us-east-1

SUMMARY
   Run an automatic debug on an application that will help highlight common/known issues.

SUPPORT
   Ask in #kubernetes
```
Note the USAGE, SUMMARY, and SUPPORT sections are static strings created by engineers.

Ideally, i could just call `WritePositionals` and `WriteFlagsForCommand` to a string writer get the output i want and plug it in where i need.